docker-compose.yml

```
@@ -0,0 +1,64 @@
+services:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)